### ISSUE TITLE
Error reading from stdin in version 0.7.2

### ISSUE THREAD

**Issue by ColDog** [garethr/kubeval#89]
Hi there, great project. We started seeing errors from kubeval on our build system this morning. The error was:

```
Could not open file -
```

We pipe yaml output into a docker container to evaluate it, this is our implementation in a nutshell:

```
cat $tmp_file | docker run -i garethr/kubeval:0.7.1 -
```

We changed the docker tag from `latest` to `0.7.1` and this fixed the image. Haven't investigated this much but it seems that something changed in `0.7.2` regarding input parsing. Let me know if you need more information!

**Comment by garethr** [comment]
Apologies for this, I didn't actually realise that would work (with the `-` as the input.) This should be fixed in #90 and I'll roll out a 0.7.3 shortly.

**Comment by ColDog** [comment]
Awesome, thanks for the quick response!
